src/v2/Apps/Artist/Routes/Overview/Components/ArtistArtworkFilter.tsx

```diff
@@ -67,6 +67,7 @@ export const ArtistArtworkFilterRefetchContainer = createRefetchContainer(
           atAuction: { type: "Boolean" }
           attributionClass: { type: "[String]" }
           color: { type: "String" }
+          colors: { type: "[String]" }
```

Comment:
Should we be removing the `color` arg? (now unused)

Reply:
Might as well